src/NServiceBus.Core/Logging/ColoredConsoleLogger.cs

```diff
@@ -1,26 +1,22 @@
 ﻿namespace NServiceBus
 {
     using System;
-    using System.Runtime.InteropServices;
     using Logging;
 
     static class ColoredConsoleLogger
     {
         static ColoredConsoleLogger()
         {
-            const int STD_OUTPUT_HANDLE = -11;
-            logToConsole = GetStdHandle(STD_OUTPUT_HANDLE) != IntPtr.Zero;
+            logToConsole = Environment.UserInteractive;
```

Comment:
That would mean it can reliably detected between Service or IIS but not differentiate between Console or WPF application. So in the worst case we would now be logging to Console in a WPF application. Would another option be to ifdef here? Thoughts?